### PR TITLE
[Windows] Fix BuildSetup.bat after #13211

### PR DESCRIPTION
**Pull request by afedchin**
@Rechi I didn't find another solution except this one.

### CODE REVIEW

File: tools/buildsteps/windows/BuildSetup.bat [hunk 1]
```diff
@@ -13,7 +13,7 @@ SET sed_exe=%msys_dir%\usr\bin\sed.exe
 
 REM read the version values from version.txt
 FOR /f %%i IN ('%awk_exe% "/APP_NAME/ {print $2}" %base_dir%\version.txt') DO SET APP_NAME=%%i
-FOR /f %%i IN ('%awk_exe% "/COMPANY_NAME/ {$1=""; print $0}" %base_dir%\version.txt') DO SET COMPANY_NAME=%%i
+FOR /f "tokens=2*" %%i IN ('findstr "COMPANY_NAME" %base_dir%\version.txt') DO SET COMPANY_NAME=%%i %%j
```
Comment (wsnipex) [inline tools/buildsteps/windows/BuildSetup.bat]: maybe sed -n '/COMPANY_NAME/ s/COMPANY_NAME *//p'

Comment (afedchin) [reply tools/buildsteps/windows/BuildSetup.bat]: thanks. this much simpler

**Comment by afedchin** [comment]
@MartijnKaijser please take a look. I've changed manifest for desktop Kodi.

**Review by MartijnKaijser — APPROVED** [review]

**Review by Rechi — APPROVED** [review]

**Comment by MartijnKaijser** [comment]
Perhaps we should name and publisher to version.txt as well?

**Comment by afedchin** [comment]
it's possible, as you wish

**Comment by afedchin** [comment]
jenkins build this please

**Comment by MartijnKaijser** [comment]
We should move away as much kodi branding as possible to version.txt if it's not too much trouble.

**Comment by MartijnKaijser** [comment]
Thx

**Pull request merged**